.github/workflows/deploy-vercel-dev.yml

```
@@ -24,14 +24,22 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Use Node.js
-        uses: actions/setup-node@v4
+        uses: actions/setup-node@v6
         with:
-          node-version: "24.11.1"
+          node-version: "24.14.0"
           cache: "yarn"
 
+      - name: Configure Registry
+        if: secrets.VERDACCIO_TOKEN != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)